src/parser/markdown.ts

```diff
@@ -17,7 +17,7 @@ mdit.use(anchor, {
     }),
 });
 /* eslint-disable @typescript-eslint/no-var-requires */
-mdit.use(require('markdown-it-emoji'));
+mdit.use(require('markdown-it-emoji').full);
```

Comment:
Probably good, though looking at this plugins README I'm not sure what the default was and how limited exactly

Reply:
Good point, I don't know either ^^ but I assume that the ✨extended✨ markdown syntax wants the full version